jsk_perception/node_scripts/aws_detect_faces.py

```diff
@@ -341,11 +349,12 @@ def image_callback(self, image):
             if pose_msgs:
                 pose_msgs.poses.append(pose_msg)
 
-        if self.use_window:
+        if visualize:
```

Comment:
This should be `self.use_window`? because we do not need to call `imshow` and when we just publish images,

Reply:
Thank you for your comments.
I modified it to separate cv2's imshow and image publishing.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2697/commits/0813d5ee7d4f583054d025e2e83c633fe1642e00